### ISSUE TITLE
[Question] Is the VNV rewrite still considered *the best FNV modlist*?

### ISSUE THREAD

**Issue by TheFrenchGhosty** [VivaNewVegas/vivanewvegas.github.io#34]
Hello, 

I've used the VNV guide in June/July of last year. 

After checking the changelog/announcement I read that the guide was (basically) rewritten in December and that the rewrite was a lot smaller because of time constraint... however it contains far less mods (for example there are 0 content mods), so my question is : is it consider better than the older one?

Do note that I don't want to add anything to the mod list, this is why I used the old one: I just want the best possible modlist in every single way, and VNV pre-rewrite was that.

I currently have to stay on my currently installed version from last year since I'm on the same play-through, but I'm curious for the future.

Basically I want to know if I should keep the installation I have *forever* (without updating anything, ever) or if the "Newer" version will one day have more mods.

**Comment by TDarkShadow** [comment]
Correct, the guide was rewritten because the earlier version of the guide, aptly named [Living In New Vegas](https://github.com/VivaNewVegas/livinginnewvegas), was way too user-error prone and we had to often troubleshoot issues. That's why the rewrite was expressly made to be more streamlined.

Also note, the guide was never intended to be "the best modlist for FNV" because that's highly subjective and every day mods get created, updated, and/or deleted. So it is almost impossible to maintain such a guide to that degree of perfection.
The guide is mostly to provide a stable Fallout New Vegas, with some quality of life features.
You can easily add mods you want to use, or modify it if you disagree with certain aspects, without corrupting your save.

You can keep the old installation but note even those mods do get updated to fix bugs, and we won't support it.

**Comment by VivaNewVegas** [comment]
just to add on, the current version of VNV is much better than Living in New Vegas. Some aspects of LINV have already become outdated due to how fast the modding community moves. It still is viable to use as a list of safe mods to add on top of current VNV tho, and that's all I'd recommend it for. 

here's the correct link for it btw: https://github.com/VivaNewVegas/Living-in-New-Vegas

**Comment by TheFrenchGhosty** [comment]
@TDarkShadow 

>the guide was never intended to be "the best modlist for FNV" because that's highly subjective and every day mods get created, updated, and/or deleted. So it is almost impossible to maintain such a guide to that degree of perfection.

I guess it was considered in the community to be the "best", but yeah obviously people have different opinions on things, and modding changes quickly.


>The guide is mostly to provide a stable Fallout New Vegas, with some quality of life features.
>You can easily add mods you want to use, or modify it if you disagree with certain aspects, without corrupting your save.

Well, yeah, but as I said "I don't want to add anything to the mod list" the modlist did everything I wanted back then: bugfix, content, visual, and gameplay. Right now it feels a bit lackluster for my taste, because it changed to be a "base" instead of a fully-featured modlist.

>You can keep the old installation but note even those mods do get updated to fix bugs, and we won't support it.

If I do keep it, I just won't update anything.

@VivaNewVegas 

>just to add on, the current version of VNV is much better than Living in New Vegas. Some aspects of LINV have already become outdated due to how fast the modding community moves.

Outdated isn't necessarily bad, back then it was perfect in my opinion.


>It still is viable to use as a list of safe mods to add on top of current VNV tho, and that's all I'd recommend it for.

As I said before: "Well, yeah, but as I said "I don't want to add anything to the mod list" the modlist did everything I wanted back then: bugfix, content, visual, and gameplay. Right now it feels a bit lackluster for my taste, because it changed to be a "base" instead of a fully-featured modlist."
